app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/SettingsScreen.kt

```diff
@@ -134,18 +144,57 @@ fun SettingsScreen(onItemClick: (SettingsOptionScreens) -> Unit, getSelectedSett
     }
 }
 
+@Composable
+private fun Theme?.getString(): String {
+    return when (this) {
+        Theme.SYSTEM -> stringResource(R.string.settingsOptionThemeSystem)
+        Theme.DARK -> stringResource(R.string.settingsOptionThemeDark)
+        Theme.LIGHT -> stringResource(R.string.settingsOptionThemeLight)
+        else -> ""
+    }
+}
+
+@Composable
+private fun ValidityPeriod?.getString(): String {
+    return this?.value?.toInt()?.let {
+        pluralStringResource(R.plurals.settingsValidityPeriodValue, it, it)
+    } ?: ""
+}
+
+@Composable
+private fun DownloadLimit?.getString() = this?.value ?: ""
+
+@Composable
+private fun EmailLanguage?.getString(): String {
+    return when (this) {
+        EmailLanguage.ENGLISH -> stringResource(R.string.settingsEmailLanguageValueEnglish)
+        EmailLanguage.FRENCH -> stringResource(R.string.settingsEmailLanguageValueFrench)
+        EmailLanguage.GERMAN -> stringResource(R.string.settingsEmailLanguageValueGerman)
+        EmailLanguage.ITALIAN -> stringResource(R.string.settingsEmailLanguageValueItalian)
+        EmailLanguage.SPANISH -> stringResource(R.string.settingsEmailLanguageValueSpanish)
+        else -> ""
+    }
+}
+
 enum class SettingsOptionScreens {
     THEME, NOTIFICATIONS,
     VALIDITY_PERIOD, DOWNLOAD_LIMIT, EMAIL_LANGUAGE,
     DISCOVER_INFOMANIAK, SHARE_IDEAS, GIVE_FEEDBACK,
 }
 
+private class DummyAppSettings(
+    override var theme: Theme = Theme.SYSTEM,
+    override var downloadLimit: DownloadLimit = DownloadLimit.TWOHUNDREDFIFTY,
```

Comment:
`DownloadLimit` and `DownloadsLimitOption` are 2 separate things but both contain the same value ?
We should factorize this.